### PR TITLE
feat(template-formula): adapt template-formula format

### PR DESCRIPTION
**Pull request by noelmcloughlin**
This PR migrates to `template-formula` format.   

This PR also includes some fixes for `deepsea.source` state (see also [upstream](https://github.com/SUSE/DeepSea/pull/1695))

### CODE REVIEW

File: docs/CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+To preserve good formatting of these notes, the following format is prescribed:
+
+* ``BREAKING CHANGE: <explanation in paragraph format>.``
+
+An example of that:
+
+.. code-block:: git
+
+   ...
+
```
Comment (myii) [inline docs/CONTRIBUTING.rst]:
```suggestion

   BREAKING CHANGE: With the removal of all of the `.sls` files under
   `template package`, this formula no longer supports the installation of
   packages.
```

Basically, want to remove everything downwards from and including the `Semantic release formulas` heading.

**Comment by noelmcloughlin** [comment]
thanks @myii. this PR is aligned to https://github.com/saltstack-formulas/sysstat-formula/pull/12 so should be good for semantic release once I fix your comments above.  This formula is only used by me and I'm happy its working good.

**Comment by noelmcloughlin** [comment]
Hi @myii travis CI is passing now (fingers crossed on suse-15 on next run - long story).  

What else is needed for a semantic release?  Do you need me to squash all the commits for example?

**Comment by noelmcloughlin** [comment]
Resolves #11

**Comment by myii** [comment]
@noelmcloughlin From a `semantic-release` perspective, everything appears ready to go.  Squashing some/all of the commits is a matter of taste; they will also affect the changelog and release notes (and that cuts both ways, too much or too little).  Minor commits (or commits that would be useless to revert) should generally be squashed but I'll leave that to your discretion.

**Comment by noelmcloughlin** [comment]
The commits are fine as is, as long as it does not make enabling semantic release more difficult.

**Comment by aboe76** [comment]
@noelmcloughlin and @myii shall I merge this one without squashing?

**Comment by noelmcloughlin** [comment]
yes I think so if @myii is happy. thanks @aboe76

**Comment by noelmcloughlin** [comment]
Hopefully Suse Travis CI is fixed too.

**Comment by myii** [comment]
@aboe76 All clear from my end; as soon as @noelmcloughlin has finished work here, it's ready to go.  From the message above, it appears that's the case.  Good work!

**Comment by noelmcloughlin** [comment]
yes. thanks guys

**Comment by myii** [comment]
Merged, thanks for all of the work @noelmcloughlin and to @aboe76 for the prompt!

**Comment by myii** [comment]
@noelmcloughlin The `Release` stage failed and I noticed that the `GH_TOKEN` was missing from Travis so I've rerun that stage after adding it.  But we've still got the write permissions issue, so I've made a request on Slack in `#formulas`.  Once that's done, we can rerun that `Release` stage and all should work fine.

**Pull request merged**